jbake-core/src/main/java/org/jbake/app/ContentStore.java

```diff
@@ -245,7 +263,7 @@ private void executeCommand(String query, Object... args) {
     }
 
     public Set<String> getAllTags() {
-        Set<String> result = new HashSet<String>();
```

Comment:
Is this an intentional change? I guess not.

Reply:
See previous answer.